src/routes/manage-selections/ManageSelectionsScreen.tsx

```diff
@@ -66,7 +66,7 @@ const ManageSelectionsScreen = ({ navigation }: ManageSelectionScreenProps): Rea
   return (
     <RouteWrapper>
       <Root contentContainerStyle={{ flexGrow: 1 }}>
-        <Heading>{getLabels().manageSelection.heading}</Heading>
+        <Heading style={{ textAlign: 'center' }}>{getLabels().manageSelection.heading}</Heading>
```

Comment:
please use `styled components` where ever its possible, f.e.
```js
const StyledHeading = styled(Heading) `
text-align: center;
`
```

Reply:
won't it change Heading globally? I need to change it only in ManageSelectionScreen